aws/resource_aws_instance.go

```diff
@@ -1644,6 +1689,18 @@ func buildAwsInstanceOpts(
 		InstanceType:          aws.String(d.Get("instance_type").(string)),
 	}
 
+	if v, ok := d.GetOk("credit_specification"); ok {
+		cs := v.([]interface{})
+		for _, csValue := range cs {
```

Comment:
Since there will only be one element if its defined, we can skip the loop and directly access `cs[0]`